gatsby-node.js

```diff
@@ -42,9 +42,17 @@ exports.onCreatePage = async ({ page, actions }) => {
 
 exports.onCreateWebpackConfig = ({ actions }) => {
   actions.setWebpackConfig({
-    node: {
-      // 'fs' fix for squid.js
-      fs: 'empty'
+    resolve: {
+      fallback: {
+        // polyfill fixes for ocean.js
```

Comment:
this one was interesting to find and affects pretty much all apps using latest webpack v5. I added fallbacks for modules where I assume we actually need them in the browser.

But ideally ocean.js users should not have to do this, and ocean.js should handle its usage of native Node.js modules (which are not available in browser) better, like using or injecting polyfills for release builds or use packages like we do with `cross-fetch` which provide polyfills by default